### DISCUSSION TITLE
How far out is the Oracle provider?

### DISCUSSION THREAD

**Discussion by rogerskk** [Mimetis/Dotmim.Sync#686] [Q&A]
I've been working on a mobile application (with SQLite db) and need to support both Oracle and MS SQL server side db's for sync.  I saw that you mentioned Oracle coming soon in the overview of the docs.  How soon can we expect that?

**Comment by Mimetis** [comment]
I have no experience in Oracle
We have an outdated PR, but this PR is too old and must be updated to be part of `DMS`
So until we found someone with enough knowledge to create a new PR / update the current PR, the Oracle provider is in stand by.

**Reply by Mimetis** [reply]
Tag @LGouellec
